packages/app-builder-lib/src/configuration.ts

```diff
@@ -201,6 +201,10 @@ export interface Configuration extends PlatformSpecificBuildOptions {
   /**
    * Appx manifest created on disk - not packed into .appx package yet.
    */
+  readonly msiProjectCreated?: ((path: string) => Promise<any> | any) | string | null
+  /**
+   * MSI project created on disk - not packed into .msi package yet.
+   */
   readonly appxManifestCreated?: ((path: string) => Promise<any> | any) | string | null
```

Comment:
Looks like the documentation got flip-flopped here
Can you please swap the two lines `appxManifestCreated` and `msiProjectCreated` so they match your docs? 🙂

Reply:
Good catch - sure...